### PR TITLE
Add amount keyword

### LINKED ISSUE #88: Feature request: Combine multiple change request activities of same PR into one line
Please check these things before submitting your issue:

- [x] I searched for duplicate or closed feature request [here](https://github.com/Readme-Workflows/recent-activity/issues?q=is%3Aissue)

**Is your feature request related to a problem? Please describe.**
Currently does the Action seem to list every change requested, even if it was submitted as one comment (See [this example](https://github.com/PlaceholderAPI/PlaceholderAPI/pull/685#pullrequestreview-719578953)) as its very own one.

**Describe the solution you'd like**
Recent-Activity should check if the PR the change request is from, is the same as previous change requests in the list.

To be more precise should the action do checks as follows:

- When finding change request, check if one was found before for the list. If not, just add to list, otherwise continue with next step.
- Check if previous action(s) for the list are also change requests. If yes, continue to next step, otherwise just add to list.
- Check if the PR is the same as with the previous change requests found. If yes, merge them all into one text.

This probably requires another text for the case of combined change requests and maybe a `{amount}` placeholder to use.
Example:
```yaml
messages:
  multiple_changes_requested: "🔴 Requested {AMOUNT} changes in {ID} in {REPO}"
```

The reason for a separate message would be, that a single change request could then be listed as `Requested one change` and not as `Requested 1 change(s)`

**Describe alternatives you've considered**
There aren't any

**Additional context**
I discovered this recently when checking my Profile README 
image

### PR DESCRIPTION
**Pull request by abhijoshi2k**
## Checks

I have...

- [x] read and understood the [Contributing Guidelines][contributing]
- [ ] Updated any necessary files such as the README.md and/or CHANGELOG.md.

## Type of Pull request

- [x] **Minor Change**  
  This Pull request doesn't break existing configuration.
- [ ] **Major Change**  
  This Pull request will break existing configuration.
- [ ] **Bug fix**  
  This Pull request will fix a (critical) bug.
- [ ] **Documentation**  
  This Pull request only changes documentation (README.md, CHANGELOG.md, etc.)
- [ ] **Other:** `__________` 

## Description

Combine multiple `changes_requested` events of PR review event into single line

Closes #88 

[contributing]: https://github.com/Readme-Workflows/.github/blob/main/.github/CONTRIBUTING.md

### CODE REVIEW

**Review by PuneetGopinath — APPROVED** [review]
I can't understand much

File: dist/index.js [hunk 1]
```diff
@@ -25169,7 +25169,7 @@ const defaultVals = {
   wiki_create: "📖 Created new wiki page {WIKI} in {REPO}",
   added_member: "🤝 Became collaborator on {REPO}",
   changes_approved: "👍 Approved {ID} in {REPO}",
-  changes_requested: "🔴 Requested changes in {ID} in {REPO}",
+  changes_requested: "🔴 Requested {AMOUNT} change(s) in {ID} in {REPO}",
```
Comment (Andre601) [inline dist/index.js:25172]: I feel like this sounds better
```suggestion
  changes_requested: "🔴 Requested {AMOUNT} change(s) for {ID} in {REPO}",
```

File: src/config.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ const defaultVals = {
   wiki_create: "📖 Created new wiki page {WIKI} in {REPO}",
   added_member: "🤝 Became collaborator on {REPO}",
   changes_approved: "👍 Approved {ID} in {REPO}",
-  changes_requested: "🔴 Requested changes in {ID} in {REPO}",
+  changes_requested: "🔴 Requested {AMOUNT} change(s) in {ID} in {REPO}",
```
Comment (Andre601) [inline src/config.js:34]: I feel like this sounds better
```suggestion
  changes_requested: "🔴 Requested {AMOUNT} change(s) for {ID} in {REPO}",
```

**Review by Andre601 — CHANGES_REQUESTED** [review]

File: dist/index.js (hunk 1 above)
Comment (abhijoshi2k) [reply dist/index.js:25172]: Cool

**Comment by abhijoshi2k** [comment]
The workflow is working as expected. Please review and merge if there are no more changes required.

**Review by Andre601 — APPROVED** [review]
I really don't know why you ask me for reviews here. I'm not a JS guy...

**Pull request merged**